package.json

```
@@ -3,28 +3,31 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@testing-library/jest-dom": "^5.16.4",
-    "@testing-library/react": "^13.2.0",
-    "@testing-library/user-event": "^13.5.0",
-    "@trustwallet/wallet-core": "^2.9.0",
-    "@types/jest": "^27.5.0",
-    "@types/node": "^16.11.34",
-    "@types/react": "^18.0.9",
-    "@types/react-dom": "^18.0.3",
+    "buffer": "^6.0.3",
+    "node-polyfill-webpack-plugin": "^1.1.4",
     "fs": "npm:empty-npm-package@1.0.0",
     "fs-extra": "^10.1.0",
-    "node-polyfill-webpack-plugin": "^1.1.4",
     "react": "^18.1.0",
     "react-app-rewired": "^2.2.1",
     "react-dom": "^18.1.0",
     "react-scripts": "5.0.1",
     "typescript": "^4.6.4",
     "web-vitals": "^2.1.4"
   },
+  "devDependencies": {
+    "@testing-library/jest-dom": "^5.16.4",
+    "@testing-library/react": "^13.2.0",
+    "@testing-library/user-event": "^13.5.0",
+    "@trustwallet/wallet-core": "^2.9.0",
+    "@types/jest": "^27.5.0",
+    "@types/node": "^16.11.34",
+    "@types/react": "^18.0.9",
+    "@types/react-dom": "^18.0.3"
+  },
   "scripts": {
     "start": "react-app-rewired start",
     "build": "react-app-rewired build",
-    "postinstall": "cp node_modules/@trustwallet/wallet-core/dist/lib/wallet-core.* public/static/js"
+    "postinstall": "mkdir -pv public/static/js && cp node_modules/@trustwallet/wallet-core/dist/lib/wallet-core.* public/static/js"
```

package.jsonsrc/initWasm.tsscripts/test.jspackage.jsonsrc/initWasm.ts85323fd945d199a530a0be3cfe4a0835819b3129scripts/test.js85323fd945d199a530a0be3cfe4a0835819b3- [ ] 129
```suggestion
    "postinstall": "mkdir -pv public/static/js && cp node_modules/@trustwallet/wallet-core/dist/lib/wallet- - # [ ] -core.* public/static/js"[]()@[]()85323fd945d199a530a0be3cfe4a0835819b3129**
```suggestion
    "postinstall": "mkdir -pv public/static/js && cp node_modules/@trustwallet/wallet-core/dist/lib/wallet-core.* public/static/js"src/App.tsx
```suggestion
    "postinstall": "mkdir -pv public/static/js && cp node_modules/@trustwallet/wallet-core/dist/lib/wallet-core.* public/static/js"85323fd945d199a530a0be3cfe4a0835819b3129

```

```
**
```